docs/sdk/src/guides/your_first_node.rs

```diff
@@ -1 +1,312 @@
 //! # Your first Node
+//!
+//! In this guide, you will learn how to run a runtime, such as the one created in
+//! [`your_first_runtime`], in a node. Within the context of this guide, we will focus on running
+//! the runtime with an [`omni-node`]. Please first read this page to learn about the OmniNode, and
+//! other options when it comes to running a node.
+//!
+//! [`your_first_runtime`] is a runtime with no consensus related code, and therefore can only be
+//! executed with a node that also expects no consensus ([`sc_consensus_manual_seal`]).
+//! `polkadot-omni-node`'s [`--dev-block-time`] precisely does this.
+//!
+//! > All of the following steps are coded as unit tests of this module. Please see `Source` of the
+//! > page for more information.
+//!
+//! ## Running The Omni Node
+//!
+//! ### Installs
+//!
+//! The `polkadot-omni-node` can either be downloaded from the latest [Release](https://github.com/paritytech/polkadot-sdk/releases/) of `polkadot-sdk`,
```

Comment:
It will be after https://github.com/paritytech/polkadot-sdk/pull/6057, I was being speculative; But good note! I will add a disclaimer about it in the page to avoid confusion.

Reply:
Speculative docs sound good! I think this can be resolved once we find out how the binaries (the referenced tooling) are released  - before/at the same time (good) or after releasing the `polkadot-sdk-docs` (speculative might not be good if the tooling is released significantly later).

Reply:
Make https://github.com/paritytech/polkadot-sdk/issues/6191 to track. But for now we have to stick to `polkadot-sdk-docs` is correct at master.